### ISSUE TITLE
[QUESTION] Error during inference with NBEATS model.

### ISSUE THREAD

**Issue by sujit25** [unit8co/darts#2642]
**Describe the issue linked to the documentation**
I have trained a time series model (NBEATS) to be more specific. Now trying to evaluate model on holdout validation set. Running into specific errors with same. Testing code is [here](https://paste.ee/p/KlOJZuin). Training code is [here](https://paste.ee/p/t68yrGdx)

**Additional context**
I have trained model with Open, high, low, close values from stock market. During evaluation also using holdout set.

**Comment by madtoinou** [comment]
Hi @sujit25,

Which lines are causing the error when you execute your code snippet?

**Comment by sujit25** [comment]
Hi @madtoinou, thanks for quick reply. I am running into issue with testing script on line 89. `prediction = model.predict(n=forecast_horizon, series=series)`. Not sure whats missing in my script. I can share test data along with model for evaluation if required.

**Comment by madtoinou** [comment]
Which version of Python and Darts are you using? I am going to try to replicate the issues, but it would be great to have more information about the shape and the content of your data (without leaking confidential information of course).

**Comment by sujit25** [comment]
I am using Python - 3.9, darts - 0.29.

**Comment by sujit-dn** [comment]
@madtoinou  Any luck with issue? I am curious to understand your analysis of issue.

**Comment by sujit25** [comment]
Hey @madtoinou, just curious if you were able to figure out issue and its root cause. I would love to hear from your about your findings.

**Comment by madtoinou** [comment]
Hi, I cannot access the training and testing code anymore. Would it be possible to post it as a message on the thread? Making it as simple as possible to simplify the debugging?

**Comment by sujit25** [comment]
@madtoinou  Sure. Let me attach here itself for reference. Adding them as text file however originally these are py files

[ts_forcaster_trainer.py.txt](https://github.com/user-attachments/files/19075596/ts_forcaster_trainer.py.txt)
[load_eval_model_test.py.txt](https://github.com/user-attachments/files/19075595/load_eval_model_test.py.txt)

**Comment by sujit-dn** [comment]
Hi @madtoinou  Any further update on issue? It seems no update on same since my last message.

**Comment by sujit25** [comment]
Hi @madtoinou, any further update on issue. I am blocked on same and cant make any progress on model improvement till its resolved.
